src/main/java/com/taboola/backstage/internal/BackstageDictionaryEndpoint.java

```diff
@@ -40,4 +40,13 @@
     @GET(BackstagePaths.BACKSTAGE_API_PATH_PREFIX + "/resources/platforms")
     @Headers("Content-Type: application/json")
     Results<Resource> getPlatforms(@Header("Authorization") String authToken) throws BackstageAPIException;
+
+    @GET(BackstagePaths.BACKSTAGE_API_PATH_PREFIX + "/resources/campaigns_properties/operating_systems")
```

Comment:
1) why there is no day of week resource representation ?
2) if I understand correctly - these endpoints will be used if the developer wants to get the resources for his own use. In addition to that, we have our own representation of them in our client to use in the campaigns pojo ?

Reply:
There is custom endpoint for everything and I am adding here only the dynamic ones.